utils/python/CIME/utils.py

```diff
@@ -0,0 +1,65 @@
+"""
+Common functions used by cime python scripts
+"""
+import logging
+import sys
+import os
```

Comment:
Some of the other stuff would be more appropriate to move into the proposed Test class - I think that we want to keep the number of routines defined outside of classes as small as possible.